composer.json

```diff
@@ -77,5 +77,11 @@
             "split-target-prefix-true": "autoload_commands",
             "split-target-prefix-false": "autoload_framework"
         }
+    },
+    "repositories": {
```

Comment:
Only add and commit the files you are working on. Strictly avoid using `git add --all` or `git add .`

Remove this file from the PR.